ci-operator/jobs/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main-presubmits.yaml

```diff
@@ -108,6 +108,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )openstack-operator-build-deploy,?($|\s.*)
```

Comment:
Minor thing, but we should disable the `build-deploy` job

Reply:
@cescgina How do we disable it?

Reply:
Set `always_run: false` in that job

Reply:
thats what i thought! thanks!